### DISCUSSION TITLE
Anytrading graph displaying backwards?

### DISCUSSION THREAD

**Discussion by dRussell68** [opendilab/DI-engine#794] [Q&A] [answered]
Hi, I'm trying to get started here, but my graph is backwards, what am I doing wrong?

![head](https://github.com/opendilab/DI-engine/assets/141889257/194bbd39-43a5-4bc6-907e-2633adbe57c6)
![graph](https://github.com/opendilab/DI-engine/assets/141889257/f5cb8a32-7104-47a2-a5cf-2367b4fd297f)

**Comment by dRussell68** [comment]
Sorry, even though I sorted the data in python, the csv file itself needed to be sorted

**Reply by Cloud-Pku** [reply, marked as answer]
I followed the steps of [colab](https://colab.research.google.com/drive/1IKfjfyb0PSyTranJemvve2V38_JUs6yB?usp=sharing#scrollTo=t1XiXK_RWatS) and the result is correct. The date 'df.head()' display is from 2009. 

Maybe you can do 'cat /usr/local/lib/python3.10/dist-packages/dizoo/gym_anytrading/envs/data/STOCKS_GOOGL.csv' to check the csv.
